frontend/src/components/CourseModal/CourseModalOverview.jsx

```diff
@@ -81,12 +82,23 @@ function CourseModalOverview({ setFilter, filter, setSeason, listing }) {
       )
     : [];
 
-  // Parse for location url and location name
-  let location_url = '';
-  for (const i in weekdays) {
-    const day = weekdays[i];
-    if (listing.times_by_day && listing.times_by_day[day]) {
-      location_url = listing.times_by_day[day][0][3];
+  const locations = new Map();
+  const times = new Map();
+  for (const day of weekdays) {
+    const info = listing.times_by_day[day];
```

Comment:
looks solid but something here makes it military time i think....

Reply:
@bearsyankees It's right: the time is now 24h. I'm not sure if that's something worth fixing as I'm personally more accustomed to 24h. If you think it's a bug, I can fix that later.